python/arcticdb/util/storage_test.py

```diff
@@ -0,0 +1,241 @@
+from arcticdb import Arctic
+import pandas as pd
+import os
+import numpy as np
+import argparse
+import re
+from datetime import datetime
+
+
+def normalize_lib_name(lib_name):
+    lib_name = lib_name.replace(".", "_")
+    lib_name = lib_name.replace("-", "_")
+
+    return lib_name
+
+
+def real_s3_credentials(shared_path: bool = True):
+    endpoint = os.getenv("ARCTICDB_REAL_S3_ENDPOINT")
+    bucket = os.getenv("ARCTICDB_REAL_S3_BUCKET")
+    region = os.getenv("ARCTICDB_REAL_S3_REGION")
+    access_key = os.getenv("ARCTICDB_REAL_S3_ACCESS_KEY")
+    secret_key = os.getenv("ARCTICDB_REAL_S3_SECRET_KEY")
+    if shared_path:
+        path_prefix = os.getenv("ARCTICDB_PERSISTENT_STORAGE_SHARED_PATH_PREFIX")
+    else:
+        path_prefix = os.getenv("ARCTICDB_PERSISTENT_STORAGE_UNIQUE_PATH_PREFIX")
+
+    clear = True if str(os.getenv("ARCTICDB_REAL_S3_CLEAR")).lower() in ["true", "1"] else False
+
+    return endpoint, bucket, region, access_key, secret_key, path_prefix, clear
+
+
+def get_real_s3_uri(shared_path: bool = True):
+    endpoint, bucket, region, access_key, secret_key, path_prefix, _ = real_s3_credentials(shared_path)
+    aws_uri = (
+        f"s3s://{endpoint}:{bucket}?access={access_key}&secret={secret_key}&region={region}&path_prefix={path_prefix}"
+    )
+    return aws_uri
+
+
+def get_seed_libraries(ac=Arctic(get_real_s3_uri())):
+    return [lib for lib in ac.list_libraries() if lib.startswith("seed_")]
+
+
+def get_test_libraries(ac=Arctic(get_real_s3_uri())):
+    return [lib for lib in ac.list_libraries() if lib.startswith("test_")]
+
+
+def test_df_3_cols(start=0):
+    return pd.DataFrame(
+        {
+            "x": np.arange(start, start + 10, dtype=np.int64),
+            "y": np.arange(start + 10, start + 20, dtype=np.int64),
+            "z": np.arange(start + 20, start + 30, dtype=np.int64),
+        },
+        index=np.arange(start, start + 10, dtype=np.int64),
+    )
+
+
+def verify_library(ac):
```

Comment:
More specific name for this and `seed_library` might be good, as they're both only useful for the little 3 col dataframes